templates/todo/api/python/todo/models.py

```diff
@@ -31,6 +31,13 @@ def __init__(self, *args, **kwargs):
     AZURE_KEY_VAULT_ENDPOINT: Optional[str] = None
     APPLICATIONINSIGHTS_CONNECTION_STRING: Optional[str] = None
     APPLICATIONINSIGHTS_ROLENAME: Optional[str] = "API"
+    PRIMARY_WRITE_KEY: str = ""
+    PRIMARY_READONLY_KEY: str = ""
+    SECONDARY_WRITE_CONNECTION_STRING: str = ""
+    PRIMARY_READONLY_CONNECTION_STRING: str = ""
+    SECONDARY_WRITE_KEY: str = ""
+    SECONDARY_READONLY_KEY: str = ""
+    SECONDARY_READONLY_CONNECTION_STRING: str = ""
```

Comment:
Because when creating `Cosmos Mongo` with AVM before, multiple secrets will be created by default. If these new variables are not defined, the deployment of the API will fail. For more details, please see the comment https://github.com/Azure/azure-dev/pull/3976#discussion_r1628902479 and https://github.com/Azure/azure-dev/pull/3976#discussion_r1630793097.
Now the code of `avm/res/document-db/database-account` has been updated, and multiple secrets will not be created by default. We will also update our code.